05/app.js

```diff
@@ -1,53 +1,77 @@
 const stats = {
-    paragraphs: {
-        'p1': 0,
-    },
-    links: {
-        '/dolor.html': 0,
-    }
+  paragraphs: {
+    p1: 0,
+  },
+  links: {
+    "/dolor.html": 0,
+  },
 };
 
 /* tutaj umieść swój kod */
 
+const pList = document.querySelectorAll("p");
+if (pList) {
+  const checkType = function (event) {
+    event.preventDefault();
 
-/* nie modyfikuj kodu poniżej, ale przeanalizuj go */
+    if (event.target.dataset.id) {
+      if (event.target.dataset.id === "p1") {
+        let counterForP = stats.paragraphs.p1;
+        stats.paragraphs.p1 = ++counterForP;
+      }
```

Comment:
To rozwiązanie będzie działać dla pierwszego paragrafu, a my mamy jeszcze pozostałe 2.
Trzeba napisać bardziej uniwersalne rozwiązanie tj. `stats.paragraphs[event.target.dataset.id ]`.
Pamiętaj, żeby sprawdzać czy takie `id` istnieje jak nie to trzeba je utworzyć :)